### DISCUSSION TITLE
小兵立大功 科學家培育「超級蛤蜊」解決水污染問題

### DISCUSSION THREAD

**Discussion by delia07152** [1101-GS4719-C/Python-Community-GS4719-C#796] [General]
https://udn.com/news/story/6897/6685625

與牡蠣相似的蛤蜊，同時也是髒水過濾器以及水質監測器，於是科學家就利用它們淨化和監測水質，但為了避免蛤蜊習慣某區域的水質，每顆蛤蜊只能服役三個月，聽起來好可愛哈哈。

**Comment by Pietra1226** [comment]
感謝分享，原來蛤蜊有這種功用，而且科學家還可以馴化牠。

**Comment by dabbie109303501** [comment]
居然可以馴化蛤蜊，太酷了，不知道這些服役完的蛤蜊能不能吃

**Comment by AnAnistired** [comment]
我之前看過有人拍影片介紹黑水虻--一種吃廚餘的昆蟲，生態防治真的超級酷，看起來真的沒有什麼缺點，又帶來經濟效益，希望政府力量能大力推行

**Comment by ching020721** [comment]
好酷，原來蛤蜊與牡蠣都可以是髒水過濾器，不是人類單方面的吸血成果，不影響蛤蜊的生命是我很佩服也嘉許的地方。

**Comment by evanchou1204** [comment]
馴化蛤蜊好可愛，希望以後可以走更多用自然來對抗自然的方法。

**Comment by ru0372572** [comment]
用超級蛤蜊監測水質超酷，而且蛤蜊壽命長達 50 年，服役完之後的蛤生還很長

**Comment by Kotori0501** [comment]
感謝分享
我最愛吃的海鮮有這個功能，又可以進化水質，真酷，但我好奇一點就是馴化完還可以吃嗎？？

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
酷誒 竟然可以馴化蛤蠣 用自然的方是來解決自然的問題 對環境真是不錯的選擇

**Comment by CPCpyt** [comment]
第一次聽說這樣的事情！感覺好新鮮(不是說蛤蜊)
不知道未來會不會馴化其他物種來改善環境。

**Comment by peggy0329** [comment]
波蘭自來水公司採用的生物監測法好聰明
透過大自然的生物來監測環境是否處在合適的狀態
反觀，如果將這個技術使用在台灣的河流中...
感覺很造成很多蛤蜊直接死亡
希望台灣在河川與環境保育上還需再進步

**Comment by ruthchen1020** [comment]
文章的意思是，蛤仔是過濾器還是監測器?
然後原來蛤仔有五十年壽命，很長壽耶。

**Comment by StevenWanglolz** [comment]
超級蛤蠣辛苦了!感謝你們的服務!

**Comment by swerfg03joe** [comment]
藉由生物特性來協助人類生活是很聰明的作法，自然界本身就充滿各種不同的生物，然而早期人類對它們的認識只停留在食物而已，這是相當可惜的，如今能活用生物特長使生活進步，實屬難能可貴，只怕動保團體再次出現抗議
